applications/serial_lte_modem/src/main.c

```diff
@@ -42,6 +44,10 @@ extern uint8_t fota_type;
 extern uint8_t fota_stage;
 extern uint8_t fota_status;
 extern int32_t fota_info;
+#if defined(CONFIG_SECURE_BOOT)
+extern uint8_t fota_is_bl;
```

Comment:
Maybe we can reuse 'fota_type", just define a new type that is understood by SLM only